### PR TITLE
Change warning panel padding

### LINKED ISSUE #747: Remove `border-left` from warning panel
Remove `border-left` from warning panel. We want warning icon to left-align with content.

_Originally posted by @jrbarnes9 in https://github.com/ONSdigital/design-system/issues/49#issuecomment-611524541_

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
Change padding on warning panel for including warning panels in the content of collapsibles

Fixes #747 

### How to review 
- Tests pass
- Collapsibles containing warning panels are spaced correctly

### CODE REVIEW

**Comment by rmccar** [comment]
[Warning panels within collapsible (Adjust spacing)](https://trello.com/c/55bdLQvf/522-warning-panels-within-collapsible-adjust-spacing)

File: src/components/collapsible/examples/collapsible-with-warning/index.njk [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        "button": {
+            "close": "Hide this",
+            "contextSuffix": "content"
+        }
+    })
+%}
+
+
+{% from "components/panel/_macro.njk" import onsPanel %}
+    <p>Are you sure you want to delete this person?</p>
```
Comment (jrbarnes9) [inline src/components/collapsible/examples/collapsible-with-warning/index.njk:17]:
```suggestion
    <p>Need to answer separately from your household?</p>
```

File: src/components/collapsible/examples/collapsible-with-warning/index.njk [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    {% call onsPanel({
+            type: "warn"
+        })
+    %}
+        <p>All of the information about this person will be deleted</p>
```
Comment (jrbarnes9) [inline src/components/collapsible/examples/collapsible-with-warning/index.njk:22]:
```suggestion
        <p>Someone in your household must still complete a census using a household access code</p>
```

File: src/components/collapsible/examples/collapsible-with-warning/index.njk [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    {% endcall %}
+
+    <p>Deleting this person will delete all information that has been entered about them</p>
```
Comment (jrbarnes9) [inline src/components/collapsible/examples/collapsible-with-warning/index.njk:25]:
```suggestion
    
```

**Pull request merged**